### ISSUE TITLE
Skip to Main Content Does not work creating severe accessibility issues

### ISSUE THREAD

**Issue by stot3** [angular/angular#53015]
### Describe the problem that you experienced

Clicking the Skip to Main Content Button does not skip to main content. 

### Enter the URL of the topic with the problem

http://angular.dev

### Describe what you were looking for in the documentation

The skip to main content button should skip to the first non-repeated element. 

### Describe the actions that led you to experience the problem

Clicking the skip to content button. 

### Describe what you want to experience that would fix the problem

The skip to main content button should skip to the first non-repeated element.

**Comment by danieljancar** [comment]
Hey @twerske,
Is there an update on what challenges are blocking this issue? How can I assist in expediting the process? Thanks 😄

**Comment by JeanMeche** [comment]
The team still need to integrate the adev site into the build process ! The app isn't runnable locally until this has been done.

**Comment by JeanMeche** [comment]
I did take a look at it, the button works fine but has no purpose on the home page. There is no content to focus on. 

Or maybe we should focus on the editor ?

**Comment by bencodezen** [comment]
@JeanMeche Agreed on your findings. Instead of the editor, what do you think of having it focus on the "Learn Angular" button which is where we primarily want people to go?

**Comment by stot3** [comment]
It needs to say skip to main content.Sent from my iPhoneOn Mar 25, 2024, at 12:32 PM, Ben Hong ***@***.***> wrote:﻿
@JeanMeche Agreed on your findings. Instead of the editor, what do you think of having it focus on the "Learn Angular" button which is where we primarily want people to go?

**Comment by stot3** [comment]
Sorry, I read that wrong. It just needs to go to the first non-repetitive element.Sent from my iPhoneOn Mar 25, 2024, at 12:32 PM, Ben Hong ***@***.***> wrote:﻿
@JeanMeche Agreed on your findings. Instead of the editor, what do you think of having it focus on the "Learn Angular" button which is where we primarily want people to go?

**Comment by bencodezen** [comment]
@stot3 I took a look at the code and it seems like the issue is that the button looks for an H1 and there doesn't seem to be on the home page. I'll try and dive deeper to see what we can do to allow users to go to content.

**Comment by bencodezen** [comment]
Closed via https://github.com/angular/angular/pull/55052
